products/Zephyr Squad/Introduction.md

```diff
@@ -0,0 +1,9 @@
+This is the **Zephyr Squad Cloud** API. If you are looking for the Server API, please refer to the documentation for more information.
+This API uses the OpenAPI specification. If you want to generate clients for different languages, please use the link above to download the API specification and use it with your preferred tools.
+For feature requests or general support, please head to our support site.
```

Comment:
would you like to link to the support site here?

Reply:
Hey @frankkilcommins, yes the link should be added but we don't have it yet. I'm currently looking with the Product Owner what should be used here